src/types.ts

```diff
@@ -570,6 +570,21 @@ export interface VaultAccountsFilter {
     assetId?: string;
 }
 
+export interface PagedVaultAccountsRequestFilters {
+    namePrefix?: string;
+    nameSuffix?: string;
+    minAmountThreshold?: number;
+    assetId?: string;
+    orderBy?: "ASC" | "DESC";
+    limit?: number;
```

Comment:
don't you want to set some upper bound limit ? what if the user specified a very large number, and it actually has large number of Vaults ?

Reply:
we do have an upper-bound limit.. it will be specified in the official api